csrc/scheduler/expr_eval_sched.cpp

```diff
@@ -19,21 +19,26 @@ bool ExprEvalScheduler::canScheduleCompileTime(Fusion* fusion) {
   if (exprs.size() != 1) {
     scheduler_debug_utils::canScheduleRejectReason(
         heuristicType(), "Fusion must contain only a single expression.");
```

Comment:
Out of curiosity, where does the requirement that there is only a single expression come from?

Reply:
This is a remnant of the design choice we made earlier during implementation of `matmul` and `linear`.

We can revisit and lift this restriction -- I don't see any reason why we cannot support it.
Will test with a few cases.